File: kattis/easy/Bijele.java
package easy;

import java.util.Scanner;

public class Bijele {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int king = scanner.nextInt();
        int queen = scanner.nextInt();
        int rooks = scanner.nextInt();
        int bishops = scanner.nextInt();
        int knights = scanner.nextInt();
        int pawns = scanner.nextInt();

        System.out.printf("%s ", king > 0 ? 1-king : king+1);
        System.out.printf("%s ", queen > 0 ? 1-queen : queen+1);
        System.out.printf("%s ", rooks > 0 ? 2-rooks : rooks+2);
        System.out.printf("%s ", bishops > 0 ? 2-bishops : bishops+2);
        System.out.printf("%s ", knights > 0 ? 2-knights : knights+2);
        System.out.printf("%s", pawns > 0 ? 8-pawns : pawns+8);
    }
}

File: kattis/easy/ccpp/bijele.cpp
//
// Created by BBJ on 03.01.2024.
//
#include <iostream>

int main() {
    int king, queen, rooks, bishops, knights, pawns;
    std::cin >> king >> queen >> rooks >> bishops >> knights >> pawns;

    printf("%d ", king > 0 ? 1-king : king+1);
    printf("%d ", queen > 0 ? 1-queen : queen+1);
    printf("%d ", rooks > 0 ? 2-rooks : rooks+2);
    printf("%d ", bishops > 0 ? 2-bishops : bishops+2);
    printf("%d ", knights > 0 ? 2-knights : knights+2);
    printf("%d ", pawns > 0 ? 8-pawns : pawns+8);

    return 0;
}